react-components/src/util/CopyTextArea.js

```diff
@@ -40,12 +41,14 @@ class CopyTextArea extends Component {
                            ref="copyTextArea"
                            value={this.props.text}
                            multiLine={true}
-                           readOnly="readonly" />
+                           readOnly="readonly"
+                           margin ="normal"
+                           style={{width: '100%'}}/>
                 {hasClipboardAPI() && (
-                    <RaisedButton id={`${this.props.debugIdPrefix}.CopyTextArea.Button`}
-                                  onClick={this.onCopyText}>{
+                    <Button variant="raised" id={`${this.props.debugIdPrefix}.CopyTextArea.Button`}
+                                  onClick={this.onCopyText} style={{padding : 2}}>{
```

Comment:
This line should be re-indented to line up with `id` above, and `style` should probably go on a new line too.